mux/scripts/measure-frames.py

```
@@ -0,0 +1,208 @@
+import statistics
+import sys
+import tempfile
+import time
+
+
+def default_socket() -> str:
+    return os.path.join(tempfile.gettempdir(), f"cmux-mux-{os.getuid()}", "main.sock")
+
+
+class Rpc:
+    def __init__(self, path: str):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)